src/actions/index.js

```diff
@@ -127,12 +127,15 @@ export const createFolder = name => {
       name
     })
 
-    let folder
     try {
-      folder = await cozy.client.files.createDirectory({
+      const folder = await cozy.client.files.createDirectory({
         name: name,
         dirID: getState().view.displayedFolder.id
       })
+      dispatch({
```

Comment:
It is not an addition, it is a move. As a variable was declared outside the `try...catch` block because this `dispatch` was outside the `try...catch` block for no reason.